pkg/deleter/deleter.go

```diff
@@ -212,6 +218,12 @@ func (d *Deleter) deletePV(pv *v1.PersistentVolume) error {
 	return d.runProcess(pv, volMode, mountPath, config)
 }
 
+func GetDefaultDeleteFn() func(pv *v1.PersistentVolume) error {
```

Comment:
Do we use an empty function?

Reply:
yes, if we're not using variadic.

Reply:
I don't think we need a function to return a function here. We could just have a func `PlaceholderFunc` or `DoNothingFunc` or even pass an anonymous func here no?